Allan690/StoreManager-V2
# #161572757 Refactoring the views to make tests pass

#### What does this PR do?
Refactors the API views to make tests to pass
#### Description of Task to be completed?
Creation of tests for the users, products, and sales endpoints
#### How should this be manually tested?
1. Clone the repo: `git clone https://github.com/Allan690/StoreManager-V2.git`
2. Create a virtual environment in the cloned repo: `virtualenv env`
3. Install requirements: `pip install -r requirements.txt`
4. Create a file and call it testdb.ini, and enter the following credentials:
`[postgresql]
host=localhost
database=stores
user=postgres
password=postgres
port=5432`
N/B: Make sure you have postgres installed in your machine
4. Run the tests: `pytests tests` or simply `nosetests` 
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
[#161572757](https://www.pivotaltracker.com/story/show/161572757)
#### Screenshots (if appropriate)
#### Questions:
